libsolidity/ast/AST.h

```diff
@@ -131,6 +131,9 @@ class SourceUnit: public ASTNode
 
 	std::vector<ASTPointer<ASTNode>> nodes() const { return m_nodes; }
 
+	/// @returns a set of referenced SourceUnits. Flattens them if @a _flatten is true.
```

Comment:
Sure, but if flatten is false, it does not return a non-flattened structure, so I think recurse is a much better fit.